core-bundle/tests/Security/TwoFactor/ProviderTest.php

```diff
@@ -128,4 +128,18 @@ public function testValidatesTheAuthenticationCode(): void
 
         $this->assertTrue($provider->validateAuthenticationCode($user, '123456'));
     }
+
+    public function testPreparesTheAuthentication(): void
+    {
+        /** @var User|MockObject $user */
+        $user = $this->mockClassWithProperties(User::class);
+
+        $renderer = $this->createMock(BackendFormRenderer::class);
+        $authenticator = $this->createMock(Authenticator::class);
+
+        $provider = new Provider($authenticator, $renderer);
+        $provider->prepareAuthentication($user);
+
+        $this->assertTrue(true);
```

Comment:
This test does not make much sense, does it? 😄 Since the `prepareAuthentication()` method does not actually do anything, we do not need to test it IMHO.

Reply:
Yep I know. I'm fine with removing.